src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isTableLoading, setIsTableLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  const isTableVisible = !isTableLoading && !error;
+  const isErrorVisible = !isTableLoading && error;
+
+  useEffect(() => {
+    setIsTableLoading(true);
+
+    getPeople()
+      .then(visiblePeople => {
+        const peopleWithParents = visiblePeople.map(
+          (person, _, serverPeople) => {
+            const mother = serverPeople.find(m => m.name === person.motherName);
+            const father = serverPeople.find(f => f.name === person.fatherName);
+
+            return {
+              ...person,
+              mother,
+              father,
+            };
+          },
+        );
+
+        setPeople(peopleWithParents);
+      })
+      .catch(() => {
```

Comment:
In the `catch` block, you should also show the `error` message that comes to this block to catch unexpected errors.
```
.catch((error) => { ...
```